documents/src/pages/elements/tooltip.md

```diff
@@ -464,3 +464,25 @@ A timer to show or hide a tooltip can be customized using the `--show-delay` and
 | --show-delay       | Time to show tooltip when mouse hovers, in milliseconds |
 | --hide-delay       | Time to hide tooltip when mouse leaves, in milliseconds |
 
+
+## Accessibility
+::a11y-intro::
+`ef-tooltip` is assigned to `role=tooltip`. Due to tooltip's nature, there are better approaches to describe or elaborate hidden content than using tooltip such as `aria-label`, `aria-labelledby`,  `aria-description`, and `aria-describedby`.
+
+
+> Note: Not all elements support these attributes. Read more at [w3.org/TR/using-aria/#label-support](https://www.w3.org/TR/using-aria/#label-support), [tpgi.com](https://www.tpgi.com/short-note-on-aria-label-aria-labelledby-and-aria-describedby/)
+
+```html
+<a href="./" aria-labelledby="logo-label"> 
```

Comment:
Does this example work with our own tooltips? E.g.

```
<ef-tooltip id="logo-label" role="tooltip" selector="#YOUR-ID">Refinitiv logo</ef-tooltip>
```

Reply:
It's not worked because it need to show tooltip by focusing and keyboard navigation. As we discuss in IP sprint that we won't work to let tooltip work by the behavior.

Reply:
I remember that, but we still should be able to do the same with ef-tooltip (show some JS on focus). Are not we?

Reply:
No, we said no at final at sprint demo meeting at sprint zero.